pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1389,6 +1391,15 @@ const (
 	EvictLeaderValueDeletePod = "delete-pod"
 )
 
+// The `Value` of PD leader transfer annotation controls the behavior when the leader count drops to zero, the valid value is one of:
```

Comment:
`leader count drops to zero` is not accurate for PD, it should be something like `the leader transferred to another member